doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -160,51 +174,34 @@ list of assertion commands is given in :ref:`Assertions`. The command
    the statement. In other words ``Proof using e`` is equivalent to
    ``Proof using Type + e`` for any declaration expression ``e``.
 
-   .. cmdv:: Proof using {+ @ident } with @tactic
-
-      Combines in a single line :cmd:`Proof with` and :cmd:`Proof using`.
-
-      .. seealso:: :ref:`tactics-implicit-automation`
+   .. todo: "All" should be in the grammar...
 
    .. cmdv:: Proof using All
```

Comment:
You can add it to the documented syntax (in `starred_ident_ref`). It cannot be added to the actual grammar because it is parsed as an ident (otherwise it would become a keyword). Please remove this `cmdv` and the additional indentation that you've put for the description list below.

Reply:
Got it.

> It cannot be added to the actual grammar because it is parsed as an ident (otherwise it would become a keyword).

Yeah, we don't want it to be a keyword, but it might work if the production with `All` appears first.

```
  starredidentref:
    [ [ i = identref -> { SsSingl i }
      | i = identref; "*" -> { SsFwdClose(SsSingl i) }
      | "Type" -> { SsType }
      | "Type"; "*" -> { SsFwdClose SsType } ]]
  ;
```

Reply:
It might indeed.